src/components/MaterialUI/Inputs/DatePicker.js

```diff
@@ -56,17 +68,25 @@ const WrappedDatePicker = ({ value, useTime, onChange, dateFormat, showTimeZone,
 
 	return (
 		<label className={classes.datePickerWrapper}>
-			<DatePicker
-				{...props}
-				dateFormat={dateFormat || (useTime ? "P p" : "P")}
-				selected={startDate}
-				onChange={date => updateDate(date)}
-				showTimeInput={useTime ?? false}
-				useTime={useTime ?? false}
-				customTimeInput={useTime ? <TimePicker showTimeZone={showTimeZone} /> : null}
-				timeInputLabel={timeInputLabel ?? ""}
-			/>
-			<Icon className={classes.calendarIcon} id="calendar-date" />
+			<Grid container item xs={10}>
```

Comment:
It's grid item of what? Please don't use container and item in different contexts that breaks readability.